lib/renderer/window-setup.ts

```diff
@@ -261,7 +261,7 @@ export const windowSetup = (
         url = resolveURL(url, location.href);
       }
       const guestId = ipcRendererInternal.sendSync(IPC_MESSAGES.GUEST_WINDOW_MANAGER_WINDOW_OPEN, url, toString(frameName), toString(features));
-      if (guestId != null) {
+      if (guestId) {
```

Comment:
Should this (and the others in this file) still be `!= null`? This will now match 0 or '' or '0', which it didn't before.

Reply:
guestId is now going to be 0 instead of `null` when missing

Reply:
actually, I've reverted this change and just set `guestId` to `null` instead